tests/features/joinup_event/event.edit.feature

```diff
@@ -4,6 +4,24 @@ Feature: "Event page" editing.
   As an owner of the event
   I need to be able to edit it.
 
+  Scenario: Add and remove map
+    Given collections:
+      | title            | logo     | banner     | state     |
+      | Stream of Dreams | logo.png | banner.jpg | validated |
```

Comment:
This is using the same name of a collection that is already used in another test. Please use a unique name so we can easily recognize failing tests.

Reply:
Changes added.